core/templates/pages/skill-editor-page/editor-tab/skill-concept-card-editor/skill-concept-card-editor.directive.ts

```diff
@@ -181,7 +181,7 @@ angular.module('oppia').directive('skillConceptCardEditor', [
           ctrl.$onInit = function() {
             $scope.skill = SkillEditorStateService.getSkill();
             $scope.dragDotsImgUrl = UrlInterpolationService.getStaticImageUrl(
-              '/general/drag_dots.png');
+              '/general/drag_dots.webp');
```

Comment:
Could we support both PNG and WebP here?

Reply:
@darksun27  ping!

Reply:
done!

Reply:
Since this variable is no longer used can you remove it, ditto for the other places.